src/components/FormNav/FormNav.js

```diff
@@ -0,0 +1,28 @@
+import NavButton from "../NavButton/NavButton";
+
+function FormNav({ handleClick, part }) {
```

Comment:
Zmieniłam nazwę propertisu 'part"
```suggestion
function FormNav({ handleClick, step }) {
```

Reply:
done